src/app/workout/hooks/plans.hook.tsx

```
@@ -0,0 +1,37 @@
+            await fetch(`api/workout/plan?id=${id}`, {method: 'DELETE'})
+            context.refreshPlans && context.refreshPlans()
+        } catch (error: any) {
+            console.log(error)
+        }
+    }
+    const handleEdit = (row: WorkoutPlan) => () => {
+        formContext.setFormState && formContext.setFormState((prev) => ({
+            ...prev,
+            selectedPlan: context.plans.find(d => d.id === row.id) || null,
+            planFormType: 'edit'
+        }))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)